src/device-api/index.ts

```diff
@@ -92,20 +88,16 @@ export class SupervisorAPI {
 		});
 	}
 
-	public async stop(options?: SupervisorAPIStopOpts): Promise<void> {
+	public async stop(): Promise<void> {
 		if (this.server != null) {
```

Comment:
Could we just make `this.server` null in order to ignore multiple calls to `stop()`, we could change the error for a warning on subsequent calls

Reply:
Sure @pipex, changed to that in https://github.com/balena-os/balena-supervisor/commit/bdc57070c873359e38c3221561cfbbd6b755813d